packages/eslint-config-react-app/jest.js

```diff
@@ -12,21 +12,62 @@
 // and we try not to use "ERROR" level at all.
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['jest', 'jest-dom', 'testing-library'],
   overrides: [
     {
       files: ['**/__tests__/**/*', '**/*.{spec,test}.*'],
       env: {
         'jest/globals': true,
       },
       // A subset of the recommended rules:
-      // https://github.com/jest-community/eslint-plugin-jest#rules
       rules: {
-        'jest/expect-expect': 'warn',
-        'jest/no-identical-title': 'warn',
-        'jest/valid-describe': 'warn',
-        'jest/valid-expect': 'warn',
-        'jest/valid-expect-in-promise': 'warn',
+        // https://github.com/jest-community/eslint-plugin-jest
+        'jest/expect-expect': 'off',
+        'jest/no-commented-out-tests': 'warn',
+        'jest/no-conditional-expect': 'error',
+        'jest/no-deprecated-functions': 'warn',
+        'jest/no-disabled-tests': 'warn',
+        'jest/no-done-callback': 'error',
+        'jest/no-export': 'error',
+        'jest/no-focused-tests': 'warn',
+        'jest/no-identical-title': 'error',
+        'jest/no-interpolation-in-snapshots': 'error',
+        'jest/no-jasmine-globals': 'error',
+        'jest/no-jest-import': 'error',
+        'jest/no-mocks-import': 'error',
+        'jest/no-standalone-expect': 'off',
+        'jest/no-test-callback': 'off',
+        'jest/no-test-prefixes': 'warn',
+        'jest/valid-describe': 'error',
+        'jest/valid-expect': 'error',
+        'jest/valid-expect-in-promise': 'error',
+        'jest/valid-title': 'warn',
```

Comment:
I think we can also turn off:
- `jest/no-commented-out-tests`
- `jest/no-deprecated-functions` (TypeScript 4.0 should also help here)
- `jest/no-disabled-tests`
- `jest/no-done-callback`
- `jest/no-export`
- `jest/no-focused-tests`
- `jest/no-test-prefixes`

I'm not sure about this, maybe @ianschmitz can chime in:
- `jest/no-jasmine-globals`

Also, I couldn't `no-test-callback`, has that been deprecated?

Reply:
For now I kept the `jest/no-jasmine-globals` rule
Let me know if it needs to be removed 🙂